### DISCUSSION TITLE
Sysmon Logs

### DISCUSSION THREAD

**Discussion by kiaraho** [Security-Onion-Solutions/securityonion#3765] [Unsupported Versions] [answered]
Has anything changed in the way SO2 ingests Sysmon logs compared to SO1? I recently noticed there are no Sysmon logs populating in Kibana. I'm using the following configuration in the `/nsm/wazuh/etc/shared/agent.conf` file on all the sensors (Wazuh managers). The `/nsm/wazuh/logs/archives/archives.json` file on the sensor shows Microsoft-Windows-Security-Auditing, Microsoft-Windows-Security-SPP, etc., but no Microsoft-Windows-Sysmon logs.

```
<agent_config os="Windows">
    <localfile>
        <location>Microsoft-Windows-Sysmon/Operational</location>
        <log_format>eventchannel</log_format>
    </localfile>
    <localfile>
        <location>Microsoft-Windows-Windows Defender/Operational</location>
        <log_format>eventchannel</log_format>
    </localfile>
</agent_config>
```
The sensors were upgraded from SO1, and the Wazuh agent and Sysmon configs remain unchanged on the endpoints.

**Comment by defensivedepth** [comment]
What version of SO2 are you running? What version of the Wazuh agents on the endpoints?

**Reply by kiaraho** [reply]
I'm running SO `2.3.40` across the entire grid, and Wazuh manager `3.13.1` on each sensor. The Wazuh agents range from `3.9.5` to `3.12.3`.

**Reply by defensivedepth** [reply]
Have you confirmed that Wazuh version spread is compatible with each other?

**Reply by kiaraho** [reply]
Yes. According to this [compatibility matrix](https://documentation.wazuh.com/3.13/installation-guide/compatibility_matrix/), the compatibility between Wazuh agent and Wazuh manager is guaranteed when the Wazuh manager has a newer or equal version than the Wazuh agent. Additionally, this issue only started when I upgraded the sensors running Wazuh manager from SO1 -> SO2.

**Comment by kiaraho** [comment, marked as answer]
Looks like copying the `/nsm/wazuh/etc/shared/agent.conf` file above to the `/nsm/wazuh/etc/shared/default/` directory on the sensors resolved this issue. Defining a [centralized configuration](https://documentation.wazuh.com/3.13/user-manual/reference/centralized-configuration.html) for the default agent group was not a hard requirement in SO1, but seems to be in SO2.
